index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio mobile version skeleton</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    
+    <!-- Header -->
+   <header class="header">
+      <div class="logo">
+        John Doe
+      </div>
```

Comment:
- [REQUIRED] Please, try to surround the `John Doe` with an  `a` tag. Since this the logo and it should be clickable to lead to home page. This way it is more intuitive for web users.